src/iris/api.py

```diff
@@ -181,23 +181,30 @@ def ts_to_sql_datetime(ts):
 }
 
 incident_query = '''SELECT %s FROM `incident`
+LEFT JOIN `incident_claim` ON `incident_claim`.`incident_id` = `incident`.`id` AND `incident_claim`.`id` =
+          (SELECT `id` FROM `incident_claim` WHERE `incident_claim`.`incident_id` = `incident`.`id` ORDER BY `time` DESC, `id` DESC LIMIT 1)
```

Comment:
EXPLAIN in MySQL is still giving me DEPENDENT SUBQUERY on lines like this. It's likely that when the incident claim table grows larger, these queries will slow down. How about something like:
```
LEFT JOIN `incident_claim` ON `incident_claim`.`incident_id` = `incident`.`id`
LEFT JOIN (SELECT `id`, MAX(`time`), `incident_id` FROM `incident_claim` GROUP BY `incident_id`) `t` ON `t`.`incident_id` = `incident`.`id`
```
Although this requires our MySQL to disable ONLY_FULL_GROUP_BY (which I believe is already the case).

Reply:
You can add another join to get MAX(id) per incident_id/time, but I'm not sure if that ends up being worth it. Maybe we try it this way and switch to joins if performance becomes an issue. Or if @fellyns could weigh in?

Alternately, since we're already using id as a tiebreaker, maybe we just get MAX(id) per incident_id, although that seems a little hacky

Reply:
Given how we shard mysql the ordinality can't really be relied on. But using it as a "tie breaker" for time should be ok - fuzzy, but ok - as long as we are deterministic it will be fine.